README.md

```diff
@@ -104,7 +111,12 @@ CREATE TABLE {your_metadata_table_name} (
 
 ### Migration
 
-#### From 1.0.6
+#### From 1.0.8 to 1.1.0
```

Comment:
Version 1.0.8 hasn't been released. So, we could have only "From 1.0.5 to 1.1.0" section